modules/n1ql/pages/n1ql-language-reference/from.adoc

```diff
@@ -161,9 +161,9 @@ collection::
 If omitted, the default collection in the bucket's default scope is used.
 
 ====
-For example, `default:{backtick}travel-sample{backtick}` indicates the default collection in the default scope in the `travel-sample` bucket in the `default` namespace.
+For example, `default:{backtick}my-db{backtick}` indicates the default collection in the default scope in the `my-db` bucket in the `default` namespace.
 
-Similarly, `default:{backtick}travel-sample{backtick}.inventory.airline` indicates the `airline` collection in the `inventory` scope in the `travel-sample` bucket in the `default` namespace.
+Similarly, `default:{backtick}my-db{backtick}.samples.airline` indicates the `airline` collection in the `samples` scope in the `my-db` bucket in the `default` namespace.
```

Comment:
Bucket, keyspace, or namespace? I think we shouldn't be using the same name of `my-db` if these concepts are changing.

Reply:
We don't really need to mention the namespace here. Only `default` can be used and it doesn't have a hyphen in it